miso-web/src/main/webapp/WEB-INF/log4j.miso.properties

```diff
@@ -34,3 +34,6 @@ log4j.appender.debugFileAppender.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss
 
 #log4j.rootLogger=INFO, stdout
 log4j.rootLogger=INFO, debugFileAppender
+
+#log4j.logger.org.hibernate.SQL=DEBUG
+#log4j.logger.org.hibernate.type=TRACE
```

Comment:
Stray commented-out properties?

Reply:
No, left intentionally so I don't have to look it up every time